src/main/java/com/conveyal/r5/analyst/TravelTimeComputer.java

```diff
@@ -289,7 +289,7 @@ public OneOriginResult computeTravelTimes() throws IOException {
                         waitTimesToStops, pathsToStops);
                 // TODO factor out identical lines in else clause
                 perTargetPropagater.travelTimeReducer = travelTimeReducer;
-                perTargetPropagater.pathWriter = new PathWriter(request, network, pathList);
+                perTargetPropagater.pathWriter = new PathWriter(request, network, pathList, egressModeLinkedDestinations.size(), transitTravelTimesToStops.length);
```

Comment:
There should already be a variable in this context called `iterations`, which should be clearer to read than transitTravelTimesToStops.length. Actually I'd prefer it be called `iterations` but that's another refactor.